src/index.js

```diff
@@ -76,6 +78,29 @@ export default {
   }
 }
 
+/**
+ *
+ * @param {Environment} env
+ * @param {*} _trigger
+ */
+function config (env, _trigger) {
+  if (env.HONEYCOMB_API_KEY) {
+    return {
+      exporter: {
+        url: 'https://api.honeycomb.io/v1/traces',
```

Comment:
Does it get sent for every request? I'm slightly worried we'll run up a large bill or get rate limited immediately....

Reply:
note that if honeycomb api key is absent it sends nothing :) which for prod, for the timebeing it is :)

Reply:
we need to have a discussion about paying for honeycomb in general before we do anything other than turn on for dev testing.